lib/app.js

```diff
@@ -160,7 +186,7 @@ const MAPPINGS = {
   },
   recentChanges: ['ds:5', 1, 2, 144, 1, 1],
   comments: {
-    path: ['ds:8', 0],
+    path: [COMMENTS_PATH, 0],
     isArray: true,
     fun: helper.extractComments
```

Comment:
there's already an extract comments helper here, and it seems to only be used by this module. I suggest that you move helper.extractComments over this module and extend it to do this selective logic instead of adding the new determineCommentsPath. 

(in general, it's preferable to use an extract `fun` to customize parsing and handle these special cases instead of adding more steps to the top level function chain)